packages/design-system/src/components/Promo/Promo.md

```diff
@@ -0,0 +1,58 @@
+### Promo
+
+## Usage
+
+### Small
+
+This is appropriate for components from 180px to 400px width. Use it in sidebars, mobile and any other contextual use, where space is the limitation.
+
+```js
+<Promo size="small" header="This example headline has 40 characters" onClose={()=>window.alert('close')} linkText="Link button" buttonText="Primary action"> A description with a <b>maximum of 100</b> characters. That usually means only one or two sentences. </Promo>
+```
+
+```js noeditor
+<ComponentHtmlMarkup>
+<Promo size="small" header="This example headline has 40 characters" onClose={()=>window.alert('close')}> A description with a <b>maximum of 100</b> characters. That usually means only one or two sentences. </Promo>
+</ComponentHtmlMarkup>
+```
+
+### Medium
+
+This is appropriate for components from 400px to around 800px width.
+
+Max text colum width is 520px
```

Comment:
Should it be the same for medium and large?

Reply:
By design - yes. Thats the difference - large has buttons on the right, medium has buttons below.

Reply:
Ok, I get it. The pattern where we have max-width of text as 520px when the component can have 1500px width itself is kinda strange